engine/src/main/java/org/entando/entando/web/dataobjectmodel/DataObjectModelController.java

```diff
@@ -76,8 +78,17 @@ public void setDataObjectModelValidator(DataObjectModelValidator dataObjectModel
 
     @RestAccessControl(permission = Permission.SUPERUSER)
     @RequestMapping(value = "/{dataModelId}", method = RequestMethod.GET, produces = MediaType.APPLICATION_JSON_VALUE)
-    public ResponseEntity<?> getDataObjectModel(@Valid @PathVariable Long dataModelId) {
-        DataModelDto dataModelDto = this.getDataObjectModelService().getDataObjectModel(dataModelId);
+    public ResponseEntity<?> getDataObjectModel(@PathVariable String dataModelId) {

```

Comment:
Add some debug logging for controller methods

Reply:
Done